### PR TITLE
feat(ssh-keygen): add completion spec

### LINKED ISSUE #1535: [ssh-keygen] missing spec

### PR DESCRIPTION
**Pull request by SerpentBytes**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Adds missing completion specs.

**What is the current behavior? (You can also link to an open issue here)**

Currently missing completion specs for `ssh-keygen` command. Closes #1535.

**What is the new behavior (if this is a feature change)?**

Implement specs for `ssh-keygen`

**Additional info:**

See `man ssh-keygen`

First time contributing during Hacktoberfest.

### CODE REVIEW

File: src/ssh-keygen.ts [hunk 1]
```diff
@@ -0,0 +1,444 @@
...
+        (option) => option !== "-l" && option !== "-v" && option !== "-f"
+      ),
+    },
+    {
+      name: "-f",
+      description: "Specifies the filename of the key file",
+      args: {
+        name: "filename",
+        description: "Filename of the key file",
+        template: "filepaths",
```
Comment (fedeci) [inline src/ssh-keygen.ts:149]: This is an argument indicating a new file item which will be created so the filepaths template is not sufficient to get the best completions
```suggestion
        template: "filepaths",
        suggestCurrentToken: true,
```

**Review by fedeci — APPROVED** [review]
Thanks for this contribution @SerpentBytes! I just have one small comment to address.

**Comment by fedeci** [comment]
Thanks!

**Comment by SerpentBytes** [comment]
Thank you for allowing me to contribute to this project. 😊

**Pull request merged**